components/card.js

```diff
@@ -1,21 +1,21 @@
-export default function Card(props) {
+export default function Card({ name, title, projectType, thumbnail, profileIcon, featureColor }) {
     return (
-        <div>
-            <img
-                src="https://og-cards.vercel.app/Lorem%20Ipsum.png?theme=light&md=1&fontSize=225px&caption=A%2520website."
-                className="rounded-xl h-40 w-52"
-            />
-            <div className="flex flex-row">
-                <div className="flex flex-col ">
-                    <h2 className="font-rubik">Lorem ipsum</h2>
+        <div className="py-4">
+            <div
+                style={{ borderColor: `${featureColor}` }}
```

Comment:
no need for a template string, remove it cause the variable can get passed it directly